tosca/core/src/main/java/nl/jads/tosca/dto/FindBugInput.java

```diff
@@ -68,4 +69,12 @@ public String getAadmid() {
     public void setAadmid(String aadmid) {
         this.aadmid = aadmid;
     }
+    
+    public String getRmid() {
```

Comment:
When a resource model is saved rmid is sent.
When an aadm is saved aadmid is sent.